index.js

```diff
@@ -1,10 +1,15 @@
 /*
-	MIT License http://www.opensource.org/licenses/mit-license.php
-	Author Tobias Koppers @sokra
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
 */
+var loaderUtils = require('loader-utils');
+
 module.exports = function(source) {
-	this.cacheable && this.cacheable();
-	var value = typeof source === "string" ? JSON.parse(source) : source;
-	this.value = [value];
-	return "module.exports = " + JSON.stringify(value) + ";";
+    this.cacheable && this.cacheable();    
+    var value = typeof source === "string" ? JSON.parse(source) : source;
+    this.value = [value];
```

Comment:
Unrelated, but this can be removed since it was needed in webpack =< v0.8.0 only